components/06-examples/beheer-basis.hbs

```diff
@@ -0,0 +1,139 @@
+{{> '@header' }}
+
+<main>
+  <div class="row">
+    <div class="col-md-12">
+      <h1>Titel van beheer-applicatie</h1>
+      <h2>Subtitel voor de meta-data</h2>
+      <hr>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-6">
+      <dl>
+        <dt>Organisator</dt>
+        <dd>Gemeente Rotterdam</dd>
+        <dt>Verzoeknummer:</dt>
+        <dd>12123497987</dd>
+        <dt>Status:</dt>
+        <dd>Open</dd>
+        <dt>Creatie datum:</dt>
+        <dd>17-12-2019</dd>
+      </dl>
+    </div>
+    <div class="col-md-6">
+      <dl>
+        <dt>Contactpersoon:</dt>
+        <dd>Jan van Veen</dd>
+        <dt>Emailadres:</dt>
+        <dd>j.veen@testmail.nl</dd>
+        <dt>Telefoonnummer:</dt>
+        <dd>06-12345678</dd>
+      </dl>
+    </div>
+    <hr>
+    <div class="row">
```

Comment:
Deze `.row` staat niet goed, deze staat in een andere `.row`: Voor `.row` geldt dat er _alleen_ kolom definities in mogen. De `hr` staat ook niet goed. Ik vermoed dat er een `</div>` mist, die heb ik nu toegevoegd